Apps/WebClient/src/Server/Services/DependentService.cs

```diff
@@ -141,5 +141,17 @@ public RequestResult<IEnumerable<DependentModel>> GetDependents(string hdId, int
 
             return result;
         }
+
+        /// <inheritdoc />
+        public RequestResult<DependentModel> Remove(string dependentHdid, string delegateHdid) {
+            DBResult<UserDelegate> dbDependent = this.userDelegateDelegate.Delete(dependentHdid, delegateHdid, true);
+            RequestResult<DependentModel> result = new RequestResult<DependentModel>()
+            {
+                ResourcePayload = new DependentModel(),
+                ResultStatus = dbDependent.Status == DBStatusCode.Deleted ? ResultType.Success : ResultType.Error,
+                ResultError = dbDependent.Status == DBStatusCode.Read ? null : new RequestResultError() { ResultMessage = dbDependent.Message, ErrorCode = ErrorTranslator.ServiceError(ErrorType.CommunicationInternal, ServiceType.Database) },
```

Comment:
Should we send back the DBStatusCode.Read or something else i.e. .Delete instead here?